src/modules/core/components/Fields/Radio/CustomRadio.tsx

```diff
@@ -10,7 +11,7 @@ import { UniversalMessageValues } from '~types/index';
 import styles from './CustomRadio.css';
 
 export interface Appearance {
-  theme?: 'primary' | 'danger';
+  theme?: 'primary' | 'danger' | 'traditional';
```

Comment:
Hahahaha I just named it `traditional` because the theme makes the CustomRadio look like what a "normal" or default radio would look like. 

If you have a better name for this theme then I'll glady change it, I just didn't wanted to waste too much time thinking about it (I was stuck for like 10 minutes thinking about a better name, but nothing came to my mind besides `traditional`).

Reply:
Yeah, this is not a good name :(

Please find something more suitable _(I don't know, maybe `fatCircles`, which while still sounding silly, has the benefit of better describing how the theme looks)_